build.gradle

```diff
@@ -92,7 +99,6 @@ sourceSets {
       runtimeClasspath += main.output
       srcDir file('src/smokeTest/java')
     }
-    resources.srcDir file('src/smokeTest/resources')
```

Comment:
Don't we need resource directory for the smoke testing?

Reply:
As of now not required. We can add when ever it is necessary.